core/sync/atomic.odin

```diff
@@ -2,44 +2,452 @@ package sync
 
 import "base:intrinsics"
 
+/*
+This procedure may lower CPU consumption or yield to a hyperthreaded twin
+processor. It's exact function is architecture specific, but the intent is to
+say that you're not doing much on a CPU.
+*/
 cpu_relax :: intrinsics.cpu_relax
```

Comment:
Doesn't it make more sense for documentation to live at the origin point (`base:intrinsics`), rather than an alias? Does that mean there will be no documentation in `intrinsics` or will this be duplicated (and thus need to both be maintained in sync)?

Reply:
I'd argue both need to be documented. The alias and the intrinsics in pkg.odin-lang.org generator.

Reply:
> or will this be duplicated

Unfortunately the state of documentation is that it needs to be duplicated if as the end user you don't want to follow constant chain of references to find the relevant information. You'll see it a lot in this PR, I duplicate information as much as I can to make sure reader gets the relevant pieces without following chains of references